website/docs/reference/global-configs/command-line-flags.md

```diff
@@ -4,60 +4,91 @@ id: "command-line-flags"
 sidebar: "Command line flags"
 ---
 
-Command line (CLI) flags immediately follow `dbt` and precede your subcommand. When set, CLI flags override environment variables and profile configs.
+For consistency, command line (CLI) flags should come right after the `dbt` prefix and its subcommands. This includes "global" flags (supported for all commands). When set, CLI flags override environment variables and profile configs. 
 
-Use this non-boolean config structure, replacing  `<THIS-CONFIG>` with the config you are enabling or disabling, `<SETTING>` with the new setting for the config, and `<SUBCOMMAND>`  with the command this config applies to:
+For example, instead of using:
+
+```bash
+dbt --no-populate-cache run
+```
+
+You should use:
+
+```bash
+dbt run --no-populate-cache
+```
+
+Historically, passing flags (such as "global flags") _before_ the subcommand is a legacy functionality that dbt Labs can remove at any time. We do not support using the same flag before and after the subcommand. 
+
+<Tabs>
+
```

Comment:
Can you add a heading and a sentence describing this boolean config? I thought it had to do with the previous paragraph and got confused. Maybe something like (not married to the title or definition):
```
## Using boolean and non-boolean flags

You can construct your commands with boolean flags to enable or disable or with non-boolean flags that use specific values, such as strings. 
```